config.example.yaml

```diff
@@ -213,6 +225,9 @@ services:
       salt: secret
       default_page_size: 250
       user_agent: RESTBase
+      internal_request_whitelist:
```

Comment:
Those are different notions of internal vs. external, and I think it's not wise to mix them. Which services to forward MediaWiki authentication details to depends on the auth scheme. For example, a signed oauth token with limited permissions might be safe to forward to some services, while forwarding a general MW cookie would not be.

We already have